controllers/greptimedbcluster/controller.go

```diff
@@ -141,11 +141,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, err
 	}
 
-	if err = cluster.Validate(); err != nil {
```

Comment:
Can we make it optional? If disable webhook, use the original code.

Reply:
Add conditions: 
```
!r.EnableAdmissionWebhook {
   cluster.Validate()
}
```